lib/core/env.dart

```diff
@@ -1,6 +1,5 @@
 abstract class Env {
-  static const String apiKey =
-      'eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI2MTNkMDY4NTA0ZjM4MTFjYjUyZDJmYjA3YTQwNDI4OCIsIm5iZiI6MTcyOTM3NjU1NC4yNzQ5NDIsInN1YiI6IjY3MTQzMGFmMmJiYmE2NWY3YjEwZmRkZSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.RkMWq6aQaP8rJO_7H4LdWJwQn69nQ-qcYWVBrkasG-o';
-  static const String baseImagePath = 'https://image.tmdb.org/t/p/w500';
-  static const String baseUrl = 'https://api.themoviedb.org/3';
+  static const String webApiKey = 'AIzaSyDeJ41HQaOXgVZ6P1gjaT3i9FhkvNRTzNQ';
```

Comment:
Why we need different api keys for each of platform?

Reply:
cause firebase has different api keys for each platform